subworkflows/nf-core/align_star.nf

```diff
@@ -0,0 +1,58 @@
+//
+// Alignment with STAR
+//
+
+//params.align_options          = [:]
```

Comment:
Leftovers from old syntax. Can go, I think?

Reply:
Good to clean up.
Did it!